syntax = "proto3";

package request_models;

message RespawnPlayerRequestModel {
	int32 player_id = 1;
}